lib/packages/core/src/api/factories/transaction/getSubscription.ts

```diff
@@ -2,8 +2,8 @@
  * Original work Copyright (c) 2018 PoC-Consortium
  * Modified work Copyright (c) 2019 Burst Apps Team
  */
-import {BurstService} from '../../../service/burstService';
-import {Subscription} from '../../../typings/subscription';
+import {BurstService} from '../../../service';
+import {Subscription} from '../../..';
```

Comment:
Is this ok?

Reply:
actually, it is.... webstorm is suggesting to shorten the imports